src/status_im/ui/components/qr_code_viewer/views.cljs

```diff
@@ -18,7 +19,7 @@
 
 (defn qr-code-viewer [{:keys [style hint-style footer-style]} value hint legend]
   {:pre [(not (nil? value))]}
-  (let [{:keys [width height]} (react/get-dimensions "window")]
+  (let [{:keys [width height]} (re-frame/subscribe [:dimensions/window])]
```

Comment:
Could be using defview?

Reply:
@jeluard i thought about that too, but not sure `:pre` (or passing fn metadata in general) is supported there.

Reply:
Good point